commonv5/ts/sonarqube/types.ts

```diff
@@ -0,0 +1,23 @@
+import Metrics from "./Metrics";
```

Comment:
This is importing `Metrics` which is actually... a custom class wrapping a list of metrics. I would have liked to drop it but as a baby step IMO it's acceptable to have it like this